server/node/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/struct/BaseStructView.java

```diff
@@ -218,28 +222,38 @@ public void onClick(ClickEvent event) {
                 updateSaveButton(false, false);
             }
         });
+
+        downloadConfigurationButton.addClickHandler(new ClickHandler() {
```

Comment:
This class represents activity of Configuration schemas page, but we are working in scope of configurations of endpoint groups.